orders.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Orders endpoint
+ *
+ * LICENSE: The buyer can free use/edit/modify this software in anyway
+ * The buyer is NOT allowed to redistribute this module in anyway or resell it
+ * or redistribute it to third party
+ *
+ * @author    ROI Hunter Easy
+ * @copyright 2019 ROI Hunter
+ * @license   EULA
+ * @version   1.0
+ * @link      https://easy.roihunter.com/
+ */
+
+include(dirname(__FILE__) . '/../../config/config.inc.php');
+include(dirname(__FILE__) . '/../../init.php');
+require_once(_PS_MODULE_DIR_ . 'roihunter/classes/auth/authentication.php');
+require_once(_PS_MODULE_DIR_ . 'roihunter/roihunter.php');
+require_once(_PS_MODULE_DIR_. 'roihunter/classes/requests/OrdersResponse.php');
+
+ROIHunterAuthenticator::getInstance()->authenticate();
+
+const DEFAULT_LIMIT = 100;
+const DEFAULT_PAGE = 0;
+if ($_SERVER['REQUEST_METHOD'] == 'GET') {
+    try {
+        $dbOrdersResult = getCompletedOrdersForTheLast3Month(getPageParameter());
```

Comment:
As mentioned in Goostav PR, according to story description we want to fetch order from last 30 days not 3 months.